snapcraft/how-to/quickstart/how-snapcraft-builds-a-snap.md

```diff
@@ -0,0 +1,90 @@
+# How Snapcraft builds snaps
+The Snapcraft tool can perform a variety of operations, including:
+
+* Snap creation operations like build, stage, prime, pack, etc.
+* Store management operations like login, register, sign-build, etc.
+* Extensions-specific operations like expand-extensions and list-extensions.
+
+These operations translate into practical tasks like:
+
+* It can build snaps locally or send remote tasks to Launchpad.
+* It allows the developer to register and log into the Snap Store.
+* It can upload snaps to the Snap Store.
+* It can promote snaps to different channels.
+
+The list of available global options and commands can be checked with one of:
+
+```bash
+snapcraft help
+snapcraft help --all
+```
+
+<h2 id='heading--snapcraft'>snapcraft.yaml</h2>
+
+Snaps are created using a build recipe defined in a file called `snapcraft.yaml`.
+
+When the snapcraft tool is executed on the command line, it will look for the file in the current project work directory, either in the top-level folder or a `snap` subdirectory. If the file is found, snapcraft will then parse its contents and progress the build toward completion.
```

Comment:
@medubelko does snapcraft support `build-aux/snap`? Is it a recommended path?

Reply:
I'm not familiar with this feature, and can't find much info about it. @mr-cal would know for certain.

With the remaining migration to RTD scheduled for the end of cycle, it's likely that much of this page will be deemed out of date and retired.